drinks_touch/database/models/sales.py

```diff
@@ -0,0 +1,13 @@
+import uuid
+
+from sqlalchemy import Column, UUID, Text, Date
+from sqlalchemy.sql import func
+
+from database.storage import Base
+
+
+class Sales(Base):
+    __tablename__ = "sales"
+    id = Column(UUID(), primary_key=True, default=uuid.uuid4)
```

Comment:
Maybe more of a theoretical question... But is it possible to have a table without a primary key? For what we want to achieve we wouldn't need one, but it would for sure be weird not to be able to uniquely select a cerain row :D